Dockerfile

```diff
@@ -1,22 +1,23 @@
-FROM ubuntu
+FROM python:3-slim
 
-# Install dependencies.
-RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-     python3-pip python3-tk texlive-latex-base latexmk git emacs vim locales
+# ARG default version is master if no BUILD_CIRQ_TAG
+ARG BUILD_CIRQ_TAG=master
 
-# Configure UTF-8 encoding.
-RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen
-ENV LANG en_US.UTF-8  
-ENV LANGUAGE en_US:en  
-ENV LC_ALL en_US.UTF-8 
+# ARG default value is no editor 
+ARG BUILD_CIRQ_EDITOR=""
+#ARG BUILD_CIRQ_EDITOR="emacs vim nano"
 
-# Make python3 default
-RUN rm -f /usr/bin/python && ln -s /usr/bin/python3 /usr/bin/python
+LABEL maintainer="Martial Rioux <martial.rioux@kloodz.com>"
```

Comment:
Will you be maintaining this dockerfile? I think we should leave it to cirq maintainers.

Reply:
Hi Vtomole, Yes we can change for cirq maintainers I will make the update.

Reply:
requested change is now in place.